doc/docs/setup.md

```diff
@@ -0,0 +1,41 @@
+# Setup
+
+## Installation
+
+### Install via pip:
+
+`pip3 install https://github.com/Lonami/pyndustric/archive/refs/heads/master.zip`
+
+or
+
+`python3 -m pip install https://github.com/Lonami/pyndustric/archive/refs/heads/master.zip`
+
+### Install from source
+
+To install from the source code (for development or something)
```

Comment:
I can see where this is coming from but I'm also not sure we should explain "how to use Python". Maybe it's better to link to setuptools documentation.

Reply:
You're right, if someone does need to do that they are probably a developer already know how to setup a module